src/components/verification-grid/verification-grid.ts

```diff
@@ -1015,37 +1296,55 @@ export class VerificationGridComponent extends WithShoelace(AbstractComponent(Li
       highlightBoxElement.style.top = `${pageY}px`;
     }
 
-    this.calculateHighlightIntersection();
+    // We mimic the selection behavior of windows explorer where the selection
+    // is always additive.
+    //
+    // Note that Windows explorer will deselect tiles if you do not hold down
+    // ctrl or shift during the initial mousedown event, but you can lift the
+    // ctrl of shift key while highlighting.
+    const options: SelectionOptions = {
+      additive: true,
+    };
+
+    const intersectingTiles = this.calculateHighlightIntersection();
+
+    // If the user drags from the right to the left, we have a "negative
+    // selection".
+    // In this case, we want to select the tiles in reverse order (in the same
+    // order that the user selected them).
+    // This is done so that the correct focus order is maintained, and any
+    // future selection order behavior is automatically implemented.
+    const isNegativeSelection = highlightWidth < 0;
+    if (isNegativeSelection) {
+      intersectingTiles.reverse();
+    }
+
+    for (const tile of intersectingTiles) {
+      this.processSelection(tile.index, options);
+    }
```

Comment:
process selection takes an array of indices, perhaps you input an array of maybe reversed indices instead of doing the outer loop here? Did I understand this right?